io.github.bruttazz.FirefoxDevFlatpak.yml

```diff
@@ -0,0 +1,92 @@
+id: io.github.bruttazz.FirefoxDevFlatpak
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: firefox
+
+build-options: 
+  build-args:
+    - --share=network
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=wayland
+  - --filesystem=host
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/man'
+  - '/share/doc'
+  - '/share/man'
+  - '/share/pkgconfig'
+  - '*.la'
+  - '*.a'
+
+modules:
+  - name: firefox-dev
+    buildsystem: simple
+    build-commands:
+      # Intended to get the latest build from firefox (the 'source' functionality won't work if we didn't specify a checksum :)
+      # neither the fiefox archive supports 'latest' tag (https://archive.mozilla.org/pub/devedition/releases)
+      # raise a pr if you managed to find a better way 
+      - wget -O firefox.tar.bz2 'https://download.mozilla.org/download/en-US/?product=firefox-devedition-latest-ssl&os=linux64&lang=en-US'
```

Comment:
build MUST be done offline. This will not work.

Reply:
Ok! sure, will make a script then..